python/lsst/pipe/analysis/analysis.py

```diff
@@ -1416,8 +1416,8 @@ def plotInputCounts(self, catalog, description, plotInfoDict, log, forcedStr=Non
             alphaCmap.set_under("r")
             alphaCmap.set_over("r")
         else:
-            log.warn("plotInputCounts: Unknown extend string for matplotlib colorbar: {:}.  Setting to "
-                     "\"neither\"".format(cbarExtend))
+            log.warning("plotInputCounts: Unknown extend string for matplotlib colorbar: {:}.  Setting to "
+                        "\"neither\"".format(cbarExtend))
```

Comment:
As a general rule, we don't pre-format log messages. We are supposed to defer formatting in case the logger decides not to emit the message. This code should therefore be something like:

```python
log.warning("plotInputCounts: Unknown extend string for matplotlib colorbar: %s.  Setting to "
            "\"neither\"", cbarExtend)
```

Reply:
Since this repo has long been slated for retirement post gen2 (but still critically important until at least that point), not to mention it will never be moved out of the `lsst-dm` git repo, we have not been updating it to strictly match current coding standards.  It's a shame you get pinged on all of the changes in this repo as I can tell your pshyce cannot rest with such blatant violations...I'll go through and clean up the pre-formatted logs if you'd like!

Reply:
Okay. Forget it then. I assumed that since someone was concerned enough to fix the deprecation warning that there was a desire to fix the log handling as well.

Reply:
I had suggested converting `warn` to `warning` to avoid further warning messages about the deprecation of `warn`. I did not comment on the formatting because I knew `pipe_analysis` is going away and won't have any observable effect due to the present "incorrect" formatting. Moreover, it is not a "production" code and is not part of `lsst_distrib`, so I think this is okay.